app/src/components/App/CommitteeInfo.jsx

```diff
@@ -30,7 +30,7 @@ class CommitteeInfo extends React.Component {
           <Icon className={_s.img} />
           <h3 className={_s.name}>{ committee.name }</h3>
         </header>
-        <div className={_s.info}>
+        <div onClick={(event) => event.stopPropagation()} className={_s.info}>
```

Comment:
I think it's cleaner to just move the onClick to the header instead. Otherwise we have to do this for all new elements in this component.